docs/api/README.md

```diff
@@ -11,15 +11,16 @@ This repository was formerly refered to as the `iam-cache-server`.
 
 ## Prepare
 
+### Development
+
 ```bash
-# development
 $ git checkout develop
 $ cp .env.dev .env
```

Comment:
We also need to mention to populate IPFS configuration

Reply:
Good point.